patches/minecraft/net/minecraft/world/entity/player/Inventory.java.patch

```diff
@@ -19,8 +19,18 @@
           if (p_36049_.m_41782_()) {
              itemstack.m_41751_(p_36049_.m_41783_().m_6426_());
           }
-@@ -229,7 +_,7 @@
+@@ -222,14 +_,16 @@
+    }
+ 
+    public void m_36068_() {
++      int slotIndex = 0; // Neo - Provide the "proper" slot index as correlated with Inventory#getItem
```

Comment:
Probably worth noting that the purpose of the patch is Forge compatibility

Reply:
Only partially - this _is_ an actual bug fix for providing the proper slot index, the forge-compat part is calling the deprecated `IForgeItemStack#onInventoryTick` method instead of calling `ItemStack#inventoryTick` directly.

Reply:
Would this be a decent time though to decide on how we want to do NeoForge comments? Basically decide on the following:
1. Do we want a space between the comment and the word Neo?
2. Do we want to do like this with a `Neo - <details>` or do we want to do something like `Neo: <details>`?
3. Do we want to use `Neo` or `NeoForge`?

Reply:
I think I like `Neo: <stuff>` as a format